backend/Wheelingful.API/Wheelingful.API.csproj

```diff
@@ -8,10 +8,12 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="FluentValidation" Version="11.9.0" />
```

Comment:
I suppose you don't need to add this package explicitly.

Reply:
Good point. It is referenced by the other lib.